baip-0002.md

```diff
@@ -1,9 +1,9 @@
     BAIP: 0002
     Title: Reform of loopholes in feed price mechanism
     Authors: cn-vote  bitshareschina@163.com
```

Comment:
The whole cn-vote committee/community authored this BAIP? Why not give credit to more users than just the proxy?